content/docs/operating/configuration.md

```diff
@@ -290,3 +294,11 @@ regex: <regex>
   (`${1}`, `${2}`, ...) in `replacement` substituted by their value.
 * `keep`: Drop targets for which `regex` does not match the concatenated `source_labels`.
 * `drop`: Drop targets for which `regex` matches the concatenated `source_labels`.
+
+### Metric Relabeling `<metric_relabel_configs>`
+
+Metric Relabeling is applied to samples as the last step before ingestion . It
+has the same configuration format and actions as Target Relabeling. Metric
+Relabeling does not apply to automatically generated timeseries such as `up`.
```

Comment:
Should we just list them all (it's just two right?). We haven't defined what automatically generated time series are, IIRC.
Also space between timeseries.